bitcoinspv/submit.go

```diff
@@ -66,27 +46,17 @@ func (r *Relayer) findFirstNewHeader(ctx context.Context, indexedBlocks []*types
 	return -1, nil
 }
 
-// createHeaderMessages splits blocks into chunks and creates header messages
-func (r *Relayer) createHeaderMessages(indexedBlocks []*types.IndexedBlock) [][]wire.BlockHeader {
-	blockChunks := breakIntoChunks(indexedBlocks, int(r.Config.HeadersChunkSize))
-	headerMsgs := make([][]wire.BlockHeader, 0, len(blockChunks))
-
-	for _, chunk := range blockChunks {
-		headerMsgs = append(headerMsgs, types.NewMsgInsertHeaders(chunk))
-	}
-
-	return headerMsgs
-}
-
-func (r *Relayer) submitHeaderMessages(ctx context.Context, headers []wire.BlockHeader) error {
+func (r *Relayer) submitHeaderMessages(ctx context.Context, chunk Chunk) error {
 	err := RetryDo(r.Config.RetrySleepDuration, r.Config.MaxRetrySleepDuration, func() error {
-		if err := r.lcClient.InsertHeaders(ctx, headers); err != nil {
+		if err := r.lcClient.InsertHeaders(ctx, chunk.Headers); err != nil {
 			return err
 		}
-		first := headers[0].BlockHash().String()
-		last := headers[len(headers)-1].BlockHash().String()
+		hs := chunk.Headers
+		firstHash := hs[0].BlockHash().String()
+		lastHash := hs[len(hs)-1].BlockHash().String()
 		r.logger.Infof(
-			"Submitted %d headers [%s ... %s] to light client", len(headers), first, last)
+			"Submitted %d headers=[%s ... %s] heights=[%d ... %d] to light client",
```

Comment:
```suggestion
			"Submitted %d header/s=[%s ... %s] heights=[%d ... %d] to light client",
```

Reply:
After bootstrap we will most of the time submit only one header. We should have it formatted nicely

Reply:
OK, will add `if` and update it